web/src/main/java/se/inera/intyg/webcert/web/service/monitoring/MonitoringLogServiceImpl.java

```diff
@@ -241,57 +519,167 @@ public void logPrivatePractitionerTermsApproved(String userId, Personnummer pers
 
     @Override
     public void logNotificationSent(String hanType, String unitId, String intygsId) {
-        logEvent(MonitoringEvent.NOTIFICATION_SENT, hanType, unitId, intygsId);
+        try (MdcCloseableMap ignored = MdcCloseableMap.builder()
+            .put(MdcLogConstants.EVENT_ACTION, toEventType(MonitoringEvent.NOTIFICATION_SENT))
+            .put(MdcLogConstants.EVENT_TYPE, MdcLogConstants.EVENT_TYPE_CHANGE)
+            .put(MdcLogConstants.EVENT_STATUS_UPDATE_TYPE, hanType)
+            .put(MdcLogConstants.EVENT_CERTIFICATE_UNIT_ID, unitId)
+            .put(MdcLogConstants.EVENT_CERTIFICATE_ID, intygsId)
+            .build()
+        ) {
+            logEvent(MonitoringEvent.NOTIFICATION_SENT, hanType, unitId, intygsId);
+        }
     }
 
     @Override
     // CHECKSTYLE:OFF ParameterNumber
     public void logStatusUpdateQueued(String certificateId, String correlationId, String logicalAddress, String certificateType,
         String certificateVersion, String eventName, LocalDateTime eventTime, String currentUser) {
-        logEvent(MonitoringEvent.STATUS_UPDATE_QUEUED, certificateId, correlationId, logicalAddress, certificateType,
-            certificateVersion, eventName, eventTime, currentUser);
+        try (MdcCloseableMap ignored = MdcCloseableMap.builder()
+            .put(MdcLogConstants.EVENT_ACTION, toEventType(MonitoringEvent.STATUS_UPDATE_QUEUED))
+            .put(MdcLogConstants.EVENT_TYPE, MdcLogConstants.EVENT_TYPE_CREATION)
+            .put(MdcLogConstants.EVENT_CERTIFICATE_ID, certificateId)
+            .put(MdcLogConstants.EVENT_STATUS_UPDATE_CORRELATION_ID, correlationId)
+            .put(MdcLogConstants.EVENT_LOGICAL_ADDRESS, logicalAddress)
+            .put(MdcLogConstants.EVENT_CERTIFICATE_TYPE, certificateType)
+            .put(MdcLogConstants.EVENT_CERTIFICATE_VERSION, certificateVersion)
+            .put(MdcLogConstants.EVENT_STATUS_UPDATE_TYPE, eventName)
+            .put(MdcLogConstants.USER_ID, currentUser)
+            .build()
+        ) {
+            logEvent(MonitoringEvent.STATUS_UPDATE_QUEUED, certificateId, correlationId, logicalAddress, certificateType,
+                certificateVersion, eventName, eventTime, currentUser);
+        }
     } // CHECKSTYLE:ON ParameterNumber
 
     @Override
     public void logArendeReceived(String intygsId, String intygsTyp, String unitHsaId, ArendeAmne amne, List<String> frageIds,
         boolean isAnswer) {
+        final var subject = amne != null ? amne.name() : NO_AMNE;
```

Comment:
Would be great if we also can add event.message.id to all logs relating to an Arende was received. We need to add it as an argument.